src/offsetRotate.js

```diff
@@ -0,0 +1,81 @@
+/* global internalScope */
+
+(function () {
+  function offsetRotateParse (input) {
+    // https://drafts.fxtf.org/motion-1/#offset-rotate-property
+
+    var convertToDegrees = internalScope.convertToDegrees;
+
+    if (input === undefined) {
+      return null;
+    }
+
+    var values = input.split(/\s+/);
+    var numValues = values.length;
+    if (numValues < 1 || numValues > 2) {
+      // Invalid number of arguments for offset-rotate
+      return undefined;
+    }
+
+    var autoProvided = false;
+    var angleProvided = false;
+    var angle = 0;
+
+    for (var i = 0; i < numValues; i++) {
+      if (values[i] === 'auto') {
+        // Check that either 'auto' or 'reverse' was provided and only once
+        if (autoProvided) {
+          return undefined;
+        }
+        autoProvided = true;
+      } else if (values[i] === 'reverse') {
+        if (autoProvided) {
+          return undefined;
+        }
+        // The 'reverse' is equivalent to 'auto 180deg'
+        autoProvided = true;
+        angle += 180;
+      } else {
+        var angleDegrees = convertToDegrees(values[i]);
+        if (angleDegrees === null) {
+          return undefined;
+        }
+        if (angleProvided) {
+          return undefined;
+        }
+        angleProvided === true;
```

Comment:
offsetAnchorTest.js has
assert.equal(internalScope.offsetPositionAnchorParse('garbage% pants%'), undefined);

offsetRotateTests.js  could have similar for offset-rotate   '10deg 20deg'